src/lib/candid/to_Idl.ts

```diff
@@ -0,0 +1,45 @@
+import { IDL } from '@dfinity/candid';
```

Comment:
See if it's nice to combine all of these functions into just toIdl

Reply:
We ought not combine the three functions into one, but we can combine toParamIdl and toReturnIdl into one function, that will also make that function more generalized so we can use it to simplify encode and decode multiple.

The reason we want toIdl and what will be called toIdlArray is because one of them will return an array always (regardless of how many items are passed) and one of them will return a single value always